### PR TITLE
Test: Run tests without setup script

### PR DESCRIPTION
**Pull request by AndreasFuchsSIT**
Fix a bug where tests would not run unless a setup script was provided.

### CODE REVIEW

**Comment by williamcroberts** [comment]
I don't understand the problem or how that change fixes it, can you elaborate please?

File: test/integration/scripts/int-test-setup.sh [hunk 1]
```diff
@@ -204,6 +204,10 @@ echo ${TPM2_PKCS11_TCTI}
 # if provided, run the test script
 if [ -z "${TSETUP_SCRIPT}" ]; then
```
Comment (williamcroberts) [inline test/integration/scripts/int-test-setup.sh]: We could simplify and reduce the duplication o
```
if -n TEST_SETUP_SCRIPT:
    then run setup script
fi

run test

**Comment by AndreasFuchsSIT** [comment]
> I don't understand the problem or how that change fixes it, can you elaborate please?

The ``--tsetup-script`` parameter is options, but if left out, the test is not run.

So either ``--tsetup-script`` should be mandatory and checked or (as I assumed) it's actually optional and tests should then be run.

I can deduplicate the test invocation if you like, shall I ?

**Comment by williamcroberts** [comment]
> > I don't understand the problem or how that change fixes it, can you elaborate please?
> 
> The `--tsetup-script` parameter is options, but if left out, the test is not run.
> 
> So either `--tsetup-script` should be mandatory and checked or (as I assumed) it's actually optional and tests should then be run.
> 
> I can deduplicate the test invocation if you like, shall I ?

yeah, I get the problem now. Please de-dup and simplify the code paths.

**Pull request merged**